.changeset/cyan-drinks-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-node': patch
+---
+
+Searching for things like "World" in "Hello World." returns no results.
```

Comment:
Recommend something like...

> Improved the quality of free text searches in LunrSearchEngine.

Or similar.  Reason being, this text gets auto-added to `CHANGELOG.md`, so framing it as what was fixed vs. what was broken is helpful for future adopters!

Reply:
@iamEAP Thanks ! Changeset updated. And it's better if you are going to add the tests. 👍

Reply:
I updated the PR to have a test for trimming and one for stemming.